gunicorn/workers/base.py

```diff
@@ -134,11 +134,14 @@ def load_wsgi(self):
 
             self.log.exception(e)
 
-            exc_type, exc_val, exc_tb = sys.exc_info()
-            self.reloader.add_extra_file(exc_val.filename)
-
-            tb_string = traceback.format_exc(exc_tb)
-            self.wsgi = util.make_fail_app(tb_string)
+            try:
+                exc_type, exc_val, exc_tb = sys.exc_info()
+                self.reloader.add_extra_file(exc_val.filename)
+
+                tb_string = traceback.format_exc(exc_tb)
+                self.wsgi = util.make_fail_app(tb_string)
+            finally:
+                del exc_tb
```

Comment:
We could use a comment to explain what `del exc_tb` does here (and perhaps a reference to this PR).

Could you also squash the commits into one?

Thanks!